modules/virt-troubleshooting-vm-import.adoc

```diff
@@ -87,6 +87,22 @@ Failed to bind volumes: provisioning failed for PVC
 ----
 +
 You must use the NFS storage class. Cinder storage is not supported. link:https://bugzilla.redhat.com/show_bug.cgi?id=1857784[(*BZ#1857784*)]
+
+ifeval::["{HCOVersion}" == "2.4.1"]
+* The following error message is displayed in the *Virtual Machines* tab of the *Virtualization* screen in the {product-title} console if there is no matching {VirtProductName} template:
++
+----
+The virtual machine could not be imported.
+VMTemplateMatchingFailed: Couldn't find matching template
+----
++
+Check the following:
+
+** The guest agent correctly reports the operating system of the source VM.
+** The source VM operating system has a matching {VirtProductName} template.
+** The `vm-import-controller` ConfigMap maps the source VM operating system to the {VirtProductName} template.
```

Comment:
Not sure what is the intention here.
If we get this error it means the existing template mapping do not include the source VM OS type.